darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -58,6 +62,9 @@ def __init__(
             PyTorch loss function used for training.
             This parameter will be ignored for probabilistic models if the ``likelihood`` parameter is specified.
             Default: ``torch.nn.MSELoss()``.
+        torch_metrics
```

Comment:
Could you copy paste the docstring into all our TorchForecastingModels (TFTModel, NBEATSModel, ...)?  Otherwise this will not be shown in the model documentation.

Reply:
Added